### ISSUE TITLE
submit with tab-enter, not tab-tab-enter

### ISSUE THREAD

**Issue by chadwhitacre** [hotsh/rstat.us#632]
The tab order is Status, Twitter, Share. It should be Status, Share, Twitter.
